### ISSUE TITLE
Geometry mesh resource is not saving its relative path

### ISSUE THREAD

**Issue by tonypan1995** [KavrakiLab/robowflex#298]
In the current implementation, when a mesh geometry is loaded, the path to its resource file is saved as an exact path not a relative path.

This could create problem when such a geometry is created on one machine, and through ROS (or some other communication system) another machine is trying to load / read this geometry. In such a case, the second machine might have the mesh file in the same relative path but a different exact path.

To avoid this a safer way might be to save the relative path of the resource file.

**Comment by zkingston** [comment]
I agree that saving the relative path is important for some applications, and it contains more information than the absolute path.
